tests/tests_test_workflow/test_integ_workflow/integ_test/test_distribution_zip.py

```diff
@@ -0,0 +1,54 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import os
+import unittest
+from unittest.mock import MagicMock, Mock, patch
+
+from test_workflow.integ_test.distribution_zip import DistributionZip
+
+
+class TestDistributionZip(unittest.TestCase):
+
+    def setUp(self) -> None:
```

Comment:
There's a bit of mixing of OpenSearch and Dashboards here. Split into 3 test classes in this file `TestDistributionZip` for common logic, and `TestDistributionZipOpenSearch` and `TestDistributionZipOpenSearchDashboards`?

Reply:
Hi DB,

I think it is too complicated that way.
OS and OSD have exactly the same way of extracting artifact as well as triggering the start cmd to certain degree.

If this is more component adding up we can definitely separate them into different classes.
This is a exact one to one copy from the tar tests as well.

Thanks.